app/3.0/docs/stackblitz.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Polymer 3.0 on Stackblitz"
+---
+
+<!-- toc -->
+
+[StackBlitz](https://stackblitz.com) is an online editor for web projects. You can use StackBlitz to prototype and preview Polymer elements.
+
+Here's a [template for Polymer 3.0 to help you get started](https://stackblitz.com/edit/start-polymer3?file=start-polymer3.js). 
+
+Please note that at the time of writing, StackBlitz requires an `index.js` and `index.html` file to be present in the root folder for all JavaScript projects. You'll need to import your app shell from `index.js`, like so:
```

Comment:
Since we mention using this to prototype elements, I think "app shell" is too specific.

Maybe "... import JavaScript modules, such as elements, from `index.js`:"